src/utils/dao-helper.js

```diff
@@ -133,11 +146,52 @@ const convertDatesInObject = (data) => {
   return data;
 };
 
+/**
+ * Converts all keys in an array of objects from snake_case or UPPERCASE to camelCase.
+ *
+ * @param {Array<Object>} rows - The array of objects with keys to be converted.
+ * @returns {Array<Object>} A new array of objects with keys converted to camelCase.
+ */
+const convertKeysToCamelCase = (rows) => rows.map((row) => Object.keys(row).reduce((acc, key) => {
+  const camelCaseKey = key.toLowerCase().replace(/_([a-z])/g, (match, letter) => letter.toUpperCase());
+  acc[camelCaseKey] = row[key];
+  return acc;
+}, {}));
```

Comment:
You can simply use this without reinventing the wheel: https://lodash.com/docs/4.17.15#camelCase

Reply:
done